detectors/node/opentelemetry-resource-detector-aws/src/detectors/AwsEcsDetector.ts

```diff
@@ -38,27 +41,65 @@ import { getEnv } from '@opentelemetry/core';
 export class AwsEcsDetector implements Detector {
   readonly CONTAINER_ID_LENGTH = 64;
   readonly DEFAULT_CGROUP_PATH = '/proc/self/cgroup';
+
   private static readFileAsync = util.promisify(fs.readFile);
 
+  private static async getUrlAsJson(url: string): Promise<any> {
```

Comment:
other private methods usually get `_` prefix  and go at the end

Reply:
Done in a034f63a4dcb878cf1c6e14ab7555b3c6186a4cf